etc/torrc.d/60_network.conf

```diff
@@ -31,7 +31,8 @@
 VirtualAddrNetwork 10.192.0.0/10
```

Comment:
From the Tor manpage at https://2019.www.torproject.org/docs/tor-manual.html.en, "When providing proxy server service to a network of computers using a tool like dns-proxy-tor, change the IPv4 network to "10.192.0.0/10" or "172.16.0.0/12" and change the IPv6 network to "[FC00::]/7". I think that means this needs to change to a `VirtualAddrNetworkIPv4` line and an additional `VirtualAddrNetworkIPv6=[FC00::]/7` line is needed here.